### PR TITLE
Build Failure lint error fix

### PR DESCRIPTION
**Pull request by ashishkcerner**
### Summary
Getting the build failure due to lint errors on main branch .

**What was changed:**
Added the fix for lint error .

**Why it was changed:**

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-badge/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+*Fixed
+  *Fixed lint errors
```
Comment (supreethmr) [inline packages/terra-badge/CHANGELOG.md:6]:
```suggestion
* Fixed
  * Fixed lint errors
```

File: packages/terra-button/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+*Fixed
+  *Fixed lint errors
```
Comment (supreethmr) [inline packages/terra-button/CHANGELOG.md:6]: Need to keep the consistent with existing lint errors

File: packages/terra-badge/CHANGELOG.md (hunk 1 above)
Comment (ashishkcerner) [reply packages/terra-badge/CHANGELOG.md]: Added https://github.com/cerner/terra-core/pull/3957/commits/c098eeb35ec9a0de67b670af4e0692a1fa5fc252

File: packages/terra-button/CHANGELOG.md (hunk 2 above)
Comment (ashishkcerner) [reply packages/terra-button/CHANGELOG.md]: Added https://github.com/cerner/terra-core/pull/3957/commits/c098eeb35ec9a0de67b670af4e0692a1fa5fc252

File: packages/terra-button/src/_mixins.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 //  Map Button Variant's Color Schemes
 // ==========================
-
+/* stylelint-disable scss/no-global-function-names */
 @mixin terra-button-variant-color($color-map) {
   background-color: map-get($color-map, background-color);
   border-color: map-get($color-map, border-color);
```
Comment (sugan2416) [inline packages/terra-button/src/_mixins.scss:6]: what happens when we use 
`@use "sass:map";`
And replace map-get with `map.get()` ?

Comment (ashishkcerner) [reply packages/terra-button/src/_mixins.scss:6]: This map-get code has been added long back around 6years for which getting lint error now ,So have just disabled this lint error .

**Review by supreethmr — APPROVED** [review]

**Review by sugan2416 — APPROVED** [review]

**Comment by nikhitasharma** [comment]
@ashishkcerner We can revert this lint disabling. I have another PR out with updated package-lock which will fix these lint errors. 
cc: @supreethmr @sugan2416

**Pull request merged**